### PR TITLE
Display "Catetory field"  in the Contacts/Configuration/Contact Tags form view

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_event/views/task_category_view.xml [hunk 1]
```diff
@@ -37,3 +37,4 @@
         </field>
     </record>
 </odoo>
+
```
Comment (rimbendhaou) [inline project_event/views/task_category_view.xml]: Why this new line?

File: project_event/views/res_partner_category_view.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <field name="model">res.partner.category</field>
+        <field name="inherit_id" ref="base.view_partner_category_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='parent_id']" position="after">
+                <field name="client_type"/>
+            </xpath>
+        </field>
+    </record>
+
+    <menuitem id="menu_res_partner_category"
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_view.xml]: not the correct action, put this menu in the res_partner_category_type view with the correct action

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/views/res_partner_category_view.xml (hunk 2 above)
Comment (jananjoy) [reply project_event/views/res_partner_category_view.xml]: @rimbendhaou  Done

File: project_event/views/res_partner_category_view.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="action_partner_category" model="ir.actions.act_window">
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_view.xml]: action_partner_category_type

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/views/res_partner_category_view.xml (hunk 3 above)
Comment (jananjoy) [reply project_event/views/res_partner_category_view.xml]: @rimbendhaou Done

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**